doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_topology.rst

```diff
@@ -0,0 +1,475 @@
+* Attributes of the manifests.
+
+The following sections define these properties for products based on the nRF54H20 SoC, and describe the default configuration.
+
+SUIT manifests supported by the nRF54H20 SoC
+********************************************
+
+In the most complex predicted case, there can be 11 manifests in total, assigned across the 3 domains.
+
+Application Domain
+==================
+
+In the application domain, the following five SUIT manifests are supported:
+
+* The root manifest
+* Three local app manifests
+* A recovery manifest
+
+Local manifests are manifests that operate on the same set of local resources.
+In most cases, using a single manifest is expected to be sufficient.
+However, this document describes what is supported for the most complex scenarios, such as an A/B approach where two executable images are managed alongside a separate manifest for non-executable resources (for example, images or other media).
+
+Radio Domain
+============
+
+In the radio domain, the following three SUIT manifests are supported:
+
+* Two local radio manifests
+* A recovery manifest
+
+Secure Domain and System Controller
+===================================
+
+The Secure Domain and the System Controller support the following manifests:
+
+* The Nordic top manifest
+* A System Controller manifest
+* A Secure Domain manifest
+
+The Secure Domain Manifest is responsible solely for the delivery and initiation of the update process for the Secure Domain Firmware and Secure Domain Recovery Firmware images.
+In the following diagrams, the Secure Domain Manifest is labeled as ``SDFW`` or ``SDFW_RECOVERY``.
+
+.. figure:: ../../../../../../../modules/doc-internal/sw_arch/suit/images/nrf54h20_suit_manifests.png
+
+   SUIT manifests supported by the nRF54H20 SoC in the most complex predicted case
+
+This diagram illustrates the most complex predicted configuration.
+You can choose a subset of the manifests supported by the nRF54H20.
+
+.. figure:: ../../../../../../../modules/doc-internal/sw_arch/suit/images/nrf54h20_suit_manifests_minimal.png
+
+   SUIT manifests enabled in nRF54H20, Application and Radio images, without recovery support
+
+This diagram illustrates the minimal predicted configuration that supports the application and radio domains.
+
+Properties associated to SUIT manifest
+**************************************
+
+.. figure:: ../../../../../../../modules/doc-internal/sw_arch/suit/images/suit_manifest_properties.png
+
+   An example of property values of the SUIT Application Local Manifest
+
+This diagram presents a set of properties associated with the SUIT manifest type, along with *example* values.
+These properties are not part of the manifest itself but are configuration parameters stored on the device, defining the boundaries for allowable manifest behavior.
+The properties covered in this document include:
+
+* The property that enables the manifest
+* The vendor ID
+* The manifest class ID
```

Comment:
Should vendor ID and manifest class ID be capitalized to be consistent with the rest of the doc?
```suggestion
* The Vendor ID (VID)
* The Manifest Class ID
```

Reply:
I don't think so, unless PMT considers them as brands